CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
 
 - Radios and checkboxes no longer get distorted when placed in a flex layout, and the sibling content has a min-content width larger than the available space
 
+### Changed
+
+- Order of line-heights in `settings/typography.$line-heights` is now in order of size. If you were using the utility classes based on these values, line-heights `1` and `2` have swapped places, as have `4` and `5`.
```

Comment:
```suggestion
- Order of line-heights in `settings/typography.$line-heights` is now in order of size. If you were using the utility classes based on these values, line-heights change `u-line-height-1` for `u-line-height-2` and `u-line-height-2` to `u-line-height-1` and change `u-line-height-4` to `u-line-height-5` and `u-line-height-5` to `u-line-height-4` since both sizes have swapped places.
```

For the backendies like me to not overlook :)

Reply:
Yup, good idea 👍 I’ve used slightly different wording and updated